actor/v2action/service_access.go

```diff
@@ -0,0 +1,225 @@
+			allWarnings = append(allWarnings, warnings...)
+			if err != nil {
+				return allWarnings, err
+			}
+
+			ccv2Warnings, err := actor.CloudControllerClient.UpdateServicePlan(plan.GUID, true)
+			allWarnings = append(allWarnings, ccv2Warnings...)
+			return allWarnings, err
+		}
+	}
+
+	return allWarnings, actionerror.ServicePlanNotFoundError{PlanName: servicePlanName, ServiceName: serviceName}
+}
+
+// EnableServiceForOrg enables access for the given service in a specific org.
+func (actor Actor) EnableServiceForOrg(serviceName, organization string) (Warnings, error) {
+	var allWarnings Warnings
+
+	services, warnings, err := actor.CloudControllerClient.GetServices(ccv2.Filter{
+		Type:     constant.LabelFilter,
+		Operator: constant.EqualOperator,
+		Values:   []string{serviceName},
+	})
+	allWarnings = append(allWarnings, warnings...)
+
+	if err != nil {
+		return allWarnings, err
+	}
+
+	if len(services) == 0 {
+		return nil, actionerror.ServiceNotFoundError{Name: serviceName}
+	}
+
+	orgs, warnings, err := actor.CloudControllerClient.GetOrganizations(ccv2.Filter{
+		Type:     constant.NameFilter,
+		Operator: constant.EqualOperator,
+		Values:   []string{organization},
+	})
+
+	allWarnings = append(allWarnings, warnings...)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	if len(orgs) == 0 {
+		return nil, actionerror.OrganizationNotFoundError{Name: organization}
+	}
+
+	servicePlans, warnings, err := actor.CloudControllerClient.GetServicePlans(ccv2.Filter{
+		Type:     constant.ServiceGUIDFilter,
+		Operator: constant.EqualOperator,
+		Values:   []string{services[0].GUID},
+	})
+
+	allWarnings = append(allWarnings, warnings...)
+	if err != nil {
+		return allWarnings, err
+	}
+
+	for _, plan := range servicePlans {
+		_, warnings, err := actor.CloudControllerClient.CreateServicePlanVisibility(plan.GUID, orgs[0].GUID)
+		allWarnings = append(allWarnings, warnings...)
+		if err != nil {
+			return allWarnings, err
+		}
+	}
+
+	return allWarnings, nil
+}
+
+// EnablePlanForOrg enables access to a specific plan of the given service in a specific org.
+func (actor Actor) EnablePlanForOrg(serviceName, servicePlanName, orgName string) (Warnings, error) {
+	var allWarnings Warnings
+	services, warnings, err := actor.CloudControllerClient.GetServices(ccv2.Filter{
+		Type:     constant.LabelFilter,
+		Operator: constant.EqualOperator,
+		Values:   []string{serviceName},
+	})
+
+	allWarnings = append(allWarnings, warnings...)
+	if err != nil {
+		return allWarnings, err
+	}
+	if len(services) == 0 {
+		return nil, actionerror.ServiceNotFoundError{Name: serviceName}
+	}
+
+	orgs, _, _ := actor.CloudControllerClient.GetOrganizations(ccv2.Filter{
+		Type:     constant.NameFilter,
+		Operator: constant.EqualOperator,
+		Values:   []string{orgName},
+	})
+
+	if len(orgs) == 0 {
+		return nil, actionerror.OrganizationNotFoundError{Name: orgName}
+	}
+
+	servicePlans, warnings, err := actor.CloudControllerClient.GetServicePlans(ccv2.Filter{
```

Comment:
This block of code:
https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_access.go#L182-L191
is duplicated in:
https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_access.go#L30
https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_access.go#L70
https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_access.go#L133

This could be abstracted as `GetServicePlansForService` and  could belong to [service_plan.go](https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_plan.go)

You could also use https://github.com/cloudfoundry/cli/blob/267b4ec10bf5034d2d09ad7f4fd48d81008b015a/actor/v2action/service_summary.go#L108 by making it public. But my personal preference is the former.